src/quic.c

```diff
@@ -1038,11 +1038,17 @@ const WOLFSSL_EVP_CIPHER* wolfSSL_quic_get_hp(WOLFSSL* ssl)
 
     switch (cipher->cipherSuite) {
 #if !defined(NO_AES) && defined(HAVE_AESGCM)
+        /* This has to be CTR even though the spec says that ECB is used for
```

Comment:
Doesn't the build macro need fixed above? `WOLFSSL_AES_COUNTER`.

Reply:
Yes it does.